public/app/features/transformers/docs/content.ts

```diff
@@ -628,6 +628,52 @@ Use this transformation to construct a matrix by specifying fields from your que
   `;
     },
   },
+  groupToNestedTable: {
+    name: 'Group to nested table',
+    getHelperDocs: function () {
+      return `
+  Use this transformation to group the data by a specified field (column) value, process calculations on each group, and have records that share the same grouped field value to be displayed in a nested table.
```

Comment:
We try to keep sentences to 25 words or less (this one is 35 words) unless there's a really good reason to make it longer. I find it hard to keep the first couple clauses of this sentence in my brain by the time I get to the end. Does it work if I break it up this way?
```suggestion
  Use this transformation to group the data by a specified field (column) value and process calculations on each group. This generates records that share the same grouped field value which are displayed in a nested table.
```

Reply:
It was definitely running on, an unfortuante habit of mine 😅. This is much better!